docs/CONTRIBUTING.md

```diff
@@ -8,24 +8,30 @@ The following is a set of guidelines for contributing to this repository. Use
 your best judgment, and feel free to propose changes to this document in a pull
 request.
 
-## Types of PRs
+> [!IMPORTANT]
+> For new userstyles, make sure to see [userstyle-creation.md](./userstyle-creation.md) after reading this.
 
-It is important to note that there are **two types of PRs** for this repository:
+## Development environment
 
-1. **New Userstyles:** Please see [userstyle-creation.md](./userstyle-creation.md).
-2. **Other improvements:** These should follow the recommendations below, to help improve the chances of a merged PR.
+This repositories uses [Deno](https://deno.com/) extensively for linting and automation. We recommend setting up Deno locally to improve your workflow — see ["Installation" - Deno Docs](https://docs.deno.com/runtime/manual/getting_started/installation). With Deno installed locally, you can run the lint script using `deno task lint` and the formatting script using `deno task format`.
+
+When editing a userstyle, we suggest setting up live reloading so your local changes can be automatically reloaded through Stylus. See ["How can I see my changes in real time?"](./tips-and-tricks.md#how-can-i-see-my-changes-in-real-time).
 
 ## Recommendations
 
-### Root Variables
+### Assessing websites
+
+Some websites are, unfortunately, simply not meant for userstyles. For example, websites that have auto-generated classes (`aeN WR baA nH oy8Mbf`, `cfb2a888`, etc.) lead to unreadable and unmaintainable userstyles — they break quickly and are difficult for contributors besides the maintainer to update/maintain.
```

Comment:
I think this could do with a small transition line that introduces the preferred way. e.g. "so how can what is the preferred method" or something like that.